### PR TITLE
front: drop tsv2 switch

### LINKED ISSUE #8059: Drop TSV2 switch
### Description and goal

To start removing all TSV1 code, we need to prevent users to use the old model

### Acceptance criteria

- [ ] TSV2 switch is removed from the user settings
- [ ] App still works

### PR DESCRIPTION
**Pull request by SharglutDev**
Operational studies and stdcm both now use tsv2 model by default

e2e tests have been adapted and some removed (see 2nd commit) in accordance with @Maymanaf 

close #8059

### CODE REVIEW

File: front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx [hunk 1]
```diff
@@ -97,40 +95,22 @@ export default function TrainSettings() {
...
-            errorMsg={t('errorMessages.mandatoryField')}
-            noMargin
-          />
-        )}
+        <InputSNCF
+          type="datetime-local"
+          label={
+            <>
+              <MdOutlineAccessTime />
+              {/* TOSO TS2 : rename trainScheduleDepartureTime key to trainScheduleStartTime everywhere */}
```
Comment (Uriel-Sautron) [inline front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx]:
```suggestion
              {/* TODO TS2 : rename trainScheduleDepartureTime key to trainScheduleStartTime everywhere */}
```

Comment (Uriel-Sautron) [reply front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx]: Typo

**Review by Uriel-Sautron — COMMENTED** [review]
Review and tested on operational studie and stdcm no problem for me.
I'll keep testing.

**Review by theocrsb — APPROVED** [review]
LGTM and tested

File: front/src/applications/stdcm/hooks/useStdcm.ts [hunk 2]
```diff
@@ -199,11 +109,7 @@ const useStdcm = () => {
 
   const launchStdcmRequest = async () => {
```
Comment (clarani) [inline front/src/applications/stdcm/hooks/useStdcm.ts]: You can even rename launchStdcmRequestV2 by launchStdcmRequest and merge the 2 functions

File: front/src/common/UserSettings.tsx [hunk 3]
```diff
@@ -89,21 +80,7 @@ const UserSettings = () => {
...
         </small>
-        <div className="col-lg-6">
-          <div className="d-flex align-items-center mt-3">
-            <SwitchSNCF
-              id="train-schedule-version-switch"
-              type={SWITCH_TYPES.switch}
-              name="train-schedule-version-switch"
-              onChange={() => {
-                dispatch(switchTrainScheduleV2Activated());
-                resetStore();
```
Comment (clarani) [inline front/src/common/UserSettings.tsx:100]: I think you can remove the function resetStore, we should not call it when toggling the stdcm switch

**Review by clarani — COMMENTED** [review]
The code LGTM, I still need to test:
- [x] Eex
- [x] stdcm
- [x] rolling stock editor

File: front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx (hunk 1 above)
Comment (SharglutDev) [reply front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx]: Done

File: front/src/applications/stdcm/hooks/useStdcm.ts (hunk 2 above)
Comment (SharglutDev) [reply front/src/applications/stdcm/hooks/useStdcm.ts]: Done

File: front/src/common/UserSettings.tsx (hunk 3 above)
Comment (SharglutDev) [reply front/src/common/UserSettings.tsx:100]: Done

**Review by clarani — COMMENTED** [review]
Code looks good to me, only found 2 bugs:
- when I select a rolling stock with heating or air conditioning and reopen the rolling stock selector, then the selected rolling stock is correct but has not the correct mode (it has standard)
- when editing a rolling stock, in the effort curves tab, there is a missing translation for the air conditioning tag

**Comment by SharglutDev** [comment]
Should be fixed :)

**Review by clarani — APPROVED** [review]
LGTM, tested :white_check_mark:

**Review by clarani — COMMENTED** [review]
Last bug: if I duplicate a rollingStock (tgv 2N2 for instance) and I edit it, there is a missing translation:
image

**Review by clarani — APPROVED** [review]

**Pull request merged**